### PR TITLE
Introduce Web3Utils class

### PR DESCRIPTION
**Pull request by saturnial**
Adds ability to verify that a contract exists at a given address via: `doesContractExistAtAddressAsync`

### CODE REVIEW

File: utils/web3_utils.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import * as promisify from "tiny-promisify";
+import * as Web3 from "web3";
+import * as Web3BetaUtils from "web3-utils";
```
Comment (nadavhollander) [inline utils/web3_utils.ts]: This import is unused, no?

File: package.json [hunk 2]
```diff
@@ -37,10 +37,12 @@
     "moment": "^2.20.1",
     "solhint": "^1.1.8",
     "solidity-sha3": "^0.4.1",
+    "tiny-promisify": "^1.0.0",
     "tslint": "^5.8.0",
     "typescript": "^2.6.1",
     "web3": "0.19.0",
     "web3-typescript-typings": "^0.7.2",
+    "web3-utils": "^1.0.0-beta.30",
```
Comment (nadavhollander) [inline package.json]: Isn't this unused?

File: utils/web3_utils.ts (hunk 1 above)
Comment (saturnial) [reply utils/web3_utils.ts]: You are indeed correct

File: package.json (hunk 2 above)
Comment (saturnial) [reply package.json]: You are correct

**Pull request merged**